src/index.js

```diff
@@ -111,10 +111,15 @@ export const createOffline = (userConfig: $Shape<Config> = {}) => {
     // create store
     const store = createStore(reducer, preloadedState, enhancer);
 
-    const baseReplaceReducer = store.replaceReducer.bind(store);
-    store.replaceReducer = function replaceReducer(nextReducer) {
-      return baseReplaceReducer(enhanceReducer(nextReducer, config));
-    };
+    if (config.enhanceReplaceReducer !== false) {
+      const baseReplaceReducer = store.replaceReducer.bind(store);
+      store.replaceReducer = function replaceReducer(nextReducer) {
+        console.warn(
```

Comment:
could we remove this?

Reply:
I added this based on a suggestion in https://github.com/redux-offline/redux-offline/pull/387#issuecomment-583167141:

> Emitting warning messages when createOffline's enhanced replace reducer, warning the user to this issue might be a good idea as well.

Happy to remove it if you'd prefer though?

Reply:
I see... Ok it's fine, I'll add a check so it's only run in development. But for that I need to do some changes.

Reply:
Do you have some reliable way to check for development at the moment? I'm guessing checking `process.env.NODE_ENV !== 'production'` would not be reliable across all environments in which this library is used?

Reply:
The problem is we're running babel (no webpack, no rollup) to transpile the code. So I'd need to test if `process.env.NODE_ENV !== 'production'` works. If you want you can help me out. Add that check and run `yarn build` then test build to see if it's working. Otherwise I can do it